cmd/pool/order.go

```diff
@@ -129,6 +132,13 @@ var sharedFlags = []cli.Flag{
 			channelTypePeerDependent,
 			channelTypeScriptEnforced),
 	},
+	cli.StringFlag{
+		Name: "auction_type",
+		Usage: fmt.Sprintf("the auction market where this offer "+
+			"must be considered in during the matching (%q, %q)",
+			auctionTypeInboundLiquidity,
+			auctionTypeOutboundLiquidity),
```

Comment:
So we are also changing the meaning of the `min_chan_amount` on the asker side, right? With the outbound liquidity type the `min_chan_amount` now means "the minimum self channel balance a bidder puts in"? Maybe we should add that to the help somehow? Or update the docs in  `doc/orders.md`?
It took me a while to figure this out from the code, so users might not know this either.

Reply:
Added to the documentation. Also, the cli now returns an error explaining what needs to be set.